packages/components/src/select-control/README.md

```diff
@@ -1,89 +1,180 @@
 # SelectControl
 
-SelectControl component is used to generate select input fields.
+SelectControl allow users to select from a single-option menu. It functions as a wrapper around the browser's native `<select>` element.
 
 
-## Usage
+![A “Link To” select with “none” selected.](https://wordpress.org/gutenberg/files/2018/12/select.png)
+
+
```

Comment:
I see you removed some extra line breaks, but I still see some here and around the images. Other than that, this is looking good. 👍

Reply:
Thanks, updated!